dev/tasks/python-wheels/github.osx.amd64.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         id: vcpkg-cache
         with:
           path: vcpkg
-          key: vcpkg-{{ macos_deployment_target }}-{{ "${{ env.VCPKG_VERSION }}" }}-{{ "${{ hashFiles('arrow/ci/vcpkg/*.patch', 'arrow/ci/vcpkg/*osx*.cmake') }}" }}
```

Comment:
hi, I am new here, trying to learn :)
Wouldn't adding the file to modify the hash be enough to generate the new key? Why are you adding the `-0`? Is there some process you expect would require to increment this number?
The change looks good to me but just to understand why is the latter addition necessary

Reply:
Just in case if we want to invalidate the cache without modifying the hashed files.